modules/scanning-images.adoc

```diff
@@ -70,68 +69,85 @@ Scanner can check for vulnerabilities in images that use the following package f
 * dpkg
 * RPM
```

Comment:
rpm is lowercase. Also yum and microdnf are just package managers. the format is rpm.

apt is also a package manager. the format is just dpkg